src/tooltip.tsx

```diff
@@ -116,6 +117,9 @@ const Tooltip: React.FC<Props> = ({
     targetLabel,
     delay = true,
     dataAttributes,
+    textAlign = 'left',
```

Comment:
I did a quick search and from the examples I saw it might be interesting to leave the center as default value. Even though it's the default of lib today it seems odd to me that it's left-aligned. This can be discussed @FernandoRRosa

Reply:
I understand that if it is 'center' it will gain a strange effect visually, I think that aligned to the left gives the feeling of a paragraph structure.